src/app/portlets/shared/dot-content-types-listing/dot-content-types.component.html

```diff
@@ -1,9 +1,10 @@
 <dot-listing-data-table
+    *ngIf="showTable"
```

Comment:
Why we need this?

Reply:
we need to have all the parameters set `actionHeaderOptions `, `paginatorExtraParams `, `columns ` so the listing render correctly and avoid the `ngOnChanges` on `dot-listing-data-table` component